### ISSUE TITLE
Column breaking out on firefox when using Contao Content Slider

### ISSUE THREAD

**Issue by dennisbohn** [ErdmannFreunde/contao-grid-bundle#10]
I am using the Grid Bundle in combination with the Contao Content Slider.

It seems there is a problem on Firefox.

https://test.office.bohn.media/

The fourth element is a content slider. Everything seems fine on Chrome and Edge.

![image](https://user-images.githubusercontent.com/87128053/168287202-a7b81ce7-48ba-4e1e-b72f-a72d280ccf8f.png)

But in Firefox (Windows, v100.0) the column breaks out when it contains a Content Slider.

![image](https://user-images.githubusercontent.com/87128053/168287342-c1144884-5c38-4a9d-957d-ecacb3218ce3.png)

**Comment by denniserdmann** [comment]
Moin Dennis,

i'm not sure, if we can prevent the slider from calculating the wrong width. We had the problem at the optimist-theme too: https://github.com/erdmannfreunde/optimist-theme-bundle/issues/5#issuecomment-726824705

May you find a general solution, otherwise at least a solution for your case.

**Comment by dennisbohn** [comment]
When I set the column to fixed 640px (see third row) or to "overflow: hidden" the slider calculates the correct width. The problem is, that I want to place the buttons outside the div while I am working with floating widths.

I'll check, if this problem happens with another grid too.

**Comment by dennisbohn** [comment]
Tested it with "Bootstrap für Contao Grid" and had the same problem.

https://test.office.bohn.media/bootstrap.html

**Comment by dennisbohn** [comment]
Giving all columns a width of 100% fixed the problem.

```css
.ce_colStart {
    width: 100%;
}
```

https://test.office.bohn.media/euf.html

**Comment by denniserdmann** [comment]
Good work. I need to check, if your solution will cause other problems with the grid as there are too many options to keep in mind.

**Comment by dennisbohn** [comment]
Damn, found something that doesnt work anymore when you set the width to 100%. :'(

- justify-center
- justify-end

When the boxes grow to 100% the content is still alignes left inside the box. I will use a helper-class to set the column with the gallery to 100%.

**Comment by dennisbohn** [comment]
Damn, found something that doesnt work anymore when you set the width to 100%. :'(

- justify-center
- justify-end

When the boxes grow to 100% the content is still aligned left inside the box. "max-width" seems to work.

```css
.ce_colStart {
    max-width: 100%;
}
````

https://test.office.bohn.media/euf-maxwidth.html

**Comment by denniserdmann** [comment]
Can you test, if the following fix is working for you aswell? https://github.com/nutshell-framework/assets/commit/fb4f615ab9b7d8e322167685c6a2d2d65e15913d#diff-0f8c92d97bbd30a26ba8610e3307d91131a6bd37a91efb1edfc23ffabc21fd55R30

**Comment by dennisbohn** [comment]
It works. Thanks!

**Comment by denniserdmann** [comment]
fixed in 1.0.8
